### PR TITLE
Fix incorrect documentation about Action Pack and Action View

### PR DESCRIPTION
**Pull request by sromelus**
### Motivation / Background

The original comment incorrectly stated that Action View is part of Action Pack, which is inaccurate. This pull request fixes that error to avoid confusion.

### Detail

This change helps clarify the separation of concerns between Rails' different components and provides more accurate information about the framework's structure.

### Checklist

Before submitting the PR make sure the following are checked:

* [x] This Pull Request is related to one change. Unrelated changes should be opened in separate PRs.
* [x] Commit message has a detailed description of what changed and why. If this PR fixes a related issue include it in the commit message. Ex: `[Fix #issue-number]`
* [x] Tests are added or updated if you fix a bug or add a feature. N/A
* [x] CHANGELOG files are updated for the changed libraries if there is a behavior change or additional feature. Minor bug fixes and documentation changes should not be included. N/A

### CODE REVIEW

File: actionpack/lib/action_controller/base.rb [hunk 1]
```diff
@@ -128,7 +128,7 @@ module ActionController
   #
   # Action Controller sends content to the user by using one of five rendering
   # methods. The most versatile and common is the rendering of a template.
-  # Included in the Action Pack is the Action View, which enables rendering of ERB
+  # Rails integrates Action View,located in rails/actionview, which enables rendering of ERB
```
Comment (skipkayhil) [inline actionpack/lib/action_controller/base.rb:131]:
```suggestion
  # Also included with \Rails is Action View, which enables rendering of ERB
```

I don't think we need to mention where its located, that would be useful to someone working on the framework but not developers reading documentation.

**Comment by skipkayhil** [comment]
Thanks for contributing to Rails 🎉

**Comment by p8** [comment]
Action View is [a dependency of Action Pack](https://github.com/rails/rails/blob/ce8a6f126beca56e3183b986e005cd7996b36813/actionpack/actionpack.gemspec#L45). 
Like Active Record, Action Pack can be used outside of Rails.
So I think the original comment was correct.

**Comment by sromelus** [comment]
I think the original text was meant for older versions of [Rails 4-0](https://github.com/rails/rails/tree/4-0-stable/actionpack/lib) when actionview/ used to be inside of actionpack/

**Pull request merged**